packages/pvl/lib/patterns.js

```diff
@@ -4,13 +4,13 @@ module.exports = {
   // Currently doesn't handle other PVL-allowed numeric notations
   // For example, based notation (`16#4B#`) or scientific notation (`1.234E2`)
   numericPatterns: [
-    /^([+\-]?\d+(\.\d*)?)$/,
+    /^([+\-]?(([1-9]\d*)|0)(\.\d*)?)$/,
   ],
 
-  // Currently only handles a subset of DateTime formats, and
-  // doesn't include date-only or time-only formats
+  // Currently handles the subset of DateTime formats specified in the PDR ICD
   dateTimePatterns: [
-    /^(\d[\d+:TZ\-]+)$/,
+    /^\d{4}-\d{2}-\d{2}(T)?(\d{2}:\d{2}:\d{2}(Z)?)?$/,
```

Comment:
These updates didn't require a unit test update?

Reply:
Yeah, the units around this fail locally: 

 test/parsing.js:156

   155: test('parsing DateTime value', (t) => {                                            
   156:   t.deepEqual(parseValue('1990-07-04T12:00'), new PVLDateTime('1990-07-04T12:00'));
   157:   t.deepEqual(parseValue('1990-07-04T12:00').value, new Date('1990-07-04T12:00')); 

  Difference:

  - PVLTextString {
  -   type: 'text string',
  -   value: '1990-07-04T12:00',
  - }
  + PVLDateTime {
  +   type: 'date time',
  +   value: Date 1990-07-04 18:00:00 UTC {},
  + }

Reply:
Also what does this have to do with 2296?

Reply:
2296 mentions PDR spec compliance, the old datetime regex was *far* too lenient, and in fact attempted to parse `006` into a date. These two regexes captured all date formatting I could find in the PDR documents on CUMULUS-2296 and CUMULUS-831.

Reply:
Ah, I see.    I have some concerns about this, but they're not germain to the PR/ticket.   👍🏻